application/actions/users.go

```diff
@@ -31,6 +32,68 @@ func usersMe(c buffalo.Context) error {
 	return c.Render(http.StatusOK, r.JSON(output))
 }
 
+// swagger:operation PUT /users/me Users UsersMe
```

Comment:
Should the last word include `Update` in it?

Reply:
Yep